### ISSUE TITLE
Fixtures dependencies is not working

### ISSUE THREAD

**Issue by leandrogehlen** [yiisoft/yii2#13529]
After `yii2-codeception` to be deprecated, to load fixture dependencies is not working.

I believe, this problem is happening, because of this:
https://github.com/yiisoft/yii2-codeception/blob/master/TestCase.php#L85

and now in codeception module this line was removed

https://github.com/Codeception/Codeception/blob/2.2/src/Codeception/Module/Yii2.php#L181

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.10
| PHP version      | 7.0.8
| Operating system |

**Comment by leandrogehlen** [comment]
I have done same tests, I changed
https://github.com/Codeception/Codeception/blob/2.2/src/Codeception/Module/Yii2.php#L300 for
```php
$fixturesStore->unloadFixtures();
$fixturesStore->loadFixtures();
```
But de interesting that the `ActiveFixture` dos not implement the  `unload` method, and the reset table
is happening in `load` method:
https://github.com/yiisoft/yii2/blob/master/framework/test/ActiveFixture.php#L78

But this way, the reset table wil be not happening in correct sequence.
To solve this adding the method `unload` in `ActiveFixture`:

```php
/**
 * @inheritdoc
 */
public function unload()
{
    parent::unload();
    $this->resetTable();
}
```

**Comment by samdark** [comment]
https://github.com/Codeception/Codeception/pull/4016

**Comment by leandrogehlen** [comment]
This problem happens when exsists foreign keys. I'll refactor the test code to contemplate this situation

**Comment by ItsReddi** [comment]
Just for info. the above workaround, ist still not working if the application exits abnormally.

**Comment by samdark** [comment]
PR was merged into Codeception. @leandrogehlen, @ItsReddi would you please check if it fixed anything?

**Comment by leandrogehlen** [comment]
Unfortunately no, this issue will be fixed with #14001

**Comment by leandrogehlen** [comment]
related Codeception/Codeception#4201
